src/main/java/uk/gov/companieshouse/controller/AppealController.java

```diff
@@ -65,6 +72,11 @@ public AppealController(AppealService appealService) {
 
             return ResponseEntity.created(location).body(id);
 
+        } catch (AppealReasonException appealReasonException) {
+            LOGGER.error(appealReasonException.getMessage() +
+                " for company number {}, penalty reference {} and user id {}",
+                    companyId, penaltyReference, userId, appealReasonException);
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
         } catch (Exception ex) {
             LOGGER.error("Unable to create appeal for company number {}, penalty reference {} and user id {}",
```

Comment:
As suggested by @CH-JonathanBlake we should make this more specific than all Exceptions. https://stackify.com/common-mistakes-handling-java-exception/. We could move this and the validation into a seperate earlier try block to group them together and it might make it more obvious where this exception could come from and improve readability.

Reply:
Thanks @CH-JonathanBlake working on it